shoes-core/spec/shoes/shared_examples/state.rb

```diff
@@ -1,7 +1,8 @@
 shared_examples_for "object with state" do
   let(:input_opts) { {state: "disabled"} }
 
-  it "should initialize" do
+  it "should initialize in the right state" do
+    expect_any_instance_of(Shoes.configuration.backend_class(described_class)).to receive(:enabled).with(false)
```

Comment:
I'd have loved for this to be tested on the GUI instance and not as `Shotgun Surgery` on any class, but I couldn't find a suitable way since the gui is constructed in initialize and then the method is straight up called.

There might be a way, now that I think about it again - stubbing the method that creates the gui the return a double. But it should be fine as is imo :)